src/jit/codegencommon.cpp

```diff
@@ -11276,6 +11276,25 @@ bool Compiler::IsRegisterPassable(GenTreePtr tree)
 {
     return IsRegisterPassable(gtGetStructHandleIfPresent(tree));
 }
+
+// Returns true if the type is returned in multiple registers
+// 
+// Arguments
+//     hClass   -  type handle
+//
+// Returns
+//     true if type is returned in multiple registers
+bool Compiler::IsMultiRegRetVal(CORINFO_CLASS_HANDLE hClass)
+{
```

Comment:
CC: first lines are:

//------------------------------------------------------------------------
// <Function name>: <Short description of the function>

see conventions.txt in the JIT directory for a template to use

---
In reply to: [58249665](https://github.com/dotnet/coreclr/pull/4042#discussion_r58249665) [](ancestors = 58249665,58249592)